### ISSUE TITLE
Windows Download Link is 404

### ISSUE THREAD

**Issue by kenmcd** [twardoch/fonttools-utils#22]
I see the Windows download has been removed.
Is there a problem with the file?
Does it not work correctly?
Or?

**Comment by jonvog** [comment]
Yeah, it would indeed be great, if it was somehow possible to reupload the package...

**Comment by bebraw** [comment]
It looks like there's a zip at https://github.com/twardoch/fonttools-opentype-feature-freezer/blob/9798eb8da8ec73bbfeb93f5850528c468e6bd255/download/OTFeatureFreezer.zip . That's before it was removed from the repository.
